sonic-net/sonic-mgmt
# [m0] Fix announce routes for m0 topo

### Description of PR


Summary:
Fixes # (issue)

### Type of change





### Back port request

### Approach
#### What is the motivation for this PR?
In the previous code, announce routes for M0 is in a similar way to T0, which was incorrect and would cause M0 to see both MX and M1 as M1.

#### How did you do it?
Add function fib_m0 with reference to fib_t1.

#### How did you verify/test it?
1. Add-topo and deploy-mg for this topo.
2. Run 'show ip bgp sum' command in DUT to see numbers of route.
3. Run some test cases (dhcp_relay/acl).

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
